### DISCUSSION TITLE
Preferences Issue - "Highlight Text Wrapped In Quotes" and "Add Highlight Color To EmphaisizedText] Not highlighting text

### DISCUSSION THREAD

**Discussion by Koldunya81** [vkbo/novelWriter#1663] [Help]
I couldn't find anything on this searching for "highlight"

The issue is what it says in the title.

On my notebook computer, running Windows 11, everything works as expected. "text in double quotes is blue" and _emphasized text is orange_

On my Windows 11 desktop computer, it does not highlight anything. No blue text "within quotes," no orange in _this text_

The desktop computer DOES show green for extra spaces.

I've verified in Preferences that both installs of novelWriter are set identically (they should be set as installed, I never changed them).

I'm lost on this one as it just makes no sense at all. Both are running the same/latest version of novelWriter.

**Comment by vkbo** [comment]
I'm converting this to a help discussion, because I don't think it is a bug. The behaviour should be identical in the app, so we need to figure out if there are something else that is different. I can open a bug report later if we do identify an issue.

**Reply by vkbo** [reply]
First thing to check: Have you added any custom colour themes to the other computer? The colour used is determined by the theme. Here I mean editor theme, not the GUI theme.

**Reply by vkbo** [reply]
Second thing to check: Are you able to locate the config files on both computers and compare them? They should be saved under `C:\Users\<USER>\AppData\Local\novelwriter\novelwriter.conf` (see the [documentation](https://novelwriter.io/docs/tech_locations.html)).

The settings you are modifying here should be at the bottom of the `[Editor]` section:
```
highlightquotes = True  <---
allowopensquote = False
allowopendquote = True
highlightemph = True  <---
stopwhenidle = True
useridletime = 300
```

**Reply by Koldunya81** [reply]
No themes/custom colors, both instances are just installs of novelWriter as it defaults. I have not made any changes to any files anywhere.

That said, I verified that C:\Users\<USER>\AppData\Local\novelwriter\novelwriter.conf is as specified (true where needed), on both computers.
Also, C:\Users\<USER>\AppData\Roaming\novelwriter\Themes is empty on both computers.

**Reply by vkbo** [reply]
Actually, `C:\Users<USER>\AppData\Roaming\novelwriter\syntax` is the relevant folder here, but I assume it is empty as well.

Could you check that the quote symbols used in the text are the exact same as specified in Preferences as quote symbols for single and double quotes? The highlighter looks for the ones specified when it applies the highlighting. That wouldn't explain the issue with emphasised text though.

Also:
* Are page titles highlighted? Or any other text highlighted? Aside from the double spaces.
* Can you try switching editor theme and see if the problem is the same for the others as well? Like "Tomorrow" or "Solarized Light"?
* Can you go to Help > About Qt in the menu and give me the Qt version?

I just ran a test myself on a Windows 11 virtual machine, and I cannot find an issue at my end. So I really don't know what could be wrong.

**Reply by vkbo** [reply]
Can you also provide a screenshot of the problem from the editor? Just in case there is something lost in communication here.

**Reply by Koldunya81** [reply]
"Can you try switching editor theme and see if the problem is the same for the others as well? Like "Tomorrow" or "Solarized Light"?"

that fixed it. I changed to Solarized Light and gasped when the text changed color lol, and when I put it back to Default Light, it showed up blue inside quotations.

I'm pretty sure what I did was change the Editor to "Grey Light" at some point. Sigh.

Thanks for the help lol...

**Reply by vkbo** [reply]
lol

well, I'm glad it works now!
